src/main/java/nl/workingtalent/bookrental/controller/LoanController.java

```diff
@@ -1,12 +1,17 @@
 package nl.workingtalent.bookrental.controller;
```

Comment:
Moet ook niet ergens een koppeling komen tussen de Reservation en de Loan?
Dat hebben we ook niet niet in de class diagram, maar is het niet raar dat er niks veranderd aan de reservation als de loan aangemaakt wordt?